runtime/compiler/z/codegen/J9TreeEvaluator.cpp

```diff
@@ -11672,145 +11672,6 @@ J9::Z::TreeEvaluator::VMarrayCheckEvaluator(TR::Node *node, TR::CodeGenerator *c
    return 0;
    }
 
-static bool inlineIsAssignableFrom(TR::Node *node, TR::CodeGenerator *cg)
```

Comment:
Can I request you to keep the old code till your changes to enable the new isAssignableFrom have been absorbed ? You can open separate PR to clean this up.